src/SortableList.js

```diff
@@ -1,4 +1,4 @@
-import React, {Component} from 'react';
+imimport React, {Component} from 'react';
```

Comment:
I haven't tested this yet, but you should fix this.
```suggestion
import React, {Component} from 'react';
```